airflow/providers/apache/hive/macros/__init__.py

```diff
@@ -0,0 +1,78 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import time  # noqa
+import uuid  # noqa
+from datetime import datetime, timedelta
+from random import random  # noqa
+from typing import Any
+
+import dateutil  # noqa
+from pendulum import DateTime
+
+from airflow.providers.apache.hive.macros import hive  # noqa
+
+
+def ds_add(ds: str, days: int) -> str:
```

Comment:
Correct me if I'm wrong but these are just functions they are not macros yet.
so right now they are not usable with Jinja...
We need to make them useable
for macros we have

https://github.com/apache/airflow/blob/aefa482051ec249f42095286a0fe4cf0eec1f6f4/airflow/plugins_manager.py#L461-L491

so we need `airflow/provider/..../macros` folder to be loaded as well ?